stl/inc/xutility

```diff
@@ -3783,19 +3783,20 @@ namespace ranges {
     // clang-format on
 
     template <input_or_output_iterator _It, sentinel_for<_It> _Se>
-    subrange(_It, _Se) -> subrange<_It, _Se>;
+    subrange(_It, _Se)->subrange<_It, _Se>;
 
     template <input_or_output_iterator _It, sentinel_for<_It> _Se>
-    subrange(_It, _Se, _Make_unsigned_like_t<iter_difference_t<_It>>) -> subrange<_It, _Se, subrange_kind::sized>;
+    subrange(_It, _Se, _Make_unsigned_like_t<iter_difference_t<_It>>)->subrange<_It, _Se, subrange_kind::sized>;
 
     template <borrowed_range _Rng>
-    subrange(_Rng &&) -> subrange<iterator_t<_Rng>, sentinel_t<_Rng>,
-        (sized_range<_Rng> || sized_sentinel_for<sentinel_t<_Rng>, iterator_t<_Rng>>) ? subrange_kind::sized
-                                                                                      : subrange_kind::unsized>;
+    subrange(_Rng &&)
+        ->subrange<iterator_t<_Rng>, sentinel_t<_Rng>,
+            (sized_range<_Rng> || sized_sentinel_for<sentinel_t<_Rng>, iterator_t<_Rng>>) ? subrange_kind::sized
+                                                                                          : subrange_kind::unsized>;
 
     template <borrowed_range _Rng>
     subrange(_Rng&&, _Make_unsigned_like_t<range_difference_t<_Rng>>)
-        -> subrange<iterator_t<_Rng>, sentinel_t<_Rng>, subrange_kind::sized>;
+        ->subrange<iterator_t<_Rng>, sentinel_t<_Rng>, subrange_kind::sized>;

```

Comment:
For whatever reason my clang-format keeps messing these up

Reply:
Are you using clang-format 10, which is the version we use?

Reply:
I needed to pass the path to the binary again to vs-code. Who knows